schema.sql

```diff
@@ -1,5 +1,10 @@
-/* Database schema to keep the structure of entire database. */
+-- creating database table
 
 CREATE TABLE animals (
-    name varchar(100)
+id INT,
+name VARCHAR(100),
```

Comment:
[ Optional Changes ]

- [ ] It would be nice if you could set the ID to be required and autoincrement so when creating an animal, one doesn't need to be passing ID for each animal created. You can use `id int NOT NULL IDENTITY(1,1)` to achieve that 👍🏽
- [ ] Also, you could add `PRIMARY KEY (id)` to set the ID as the primary key for each animal created. 👍🏽